src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.props

```diff
@@ -32,6 +32,8 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
   <PropertyGroup>
     <EnableDefaultCompileItems Condition=" '$(EnableDefaultCompileItems)' == '' ">false</EnableDefaultCompileItems>                                <!--- Do not glob F# source files -->
     <EnableDefaultNoneItems Condition=" '$(EnableDefaultNoneItems)' == '' ">false</EnableDefaultNoneItems>
+    <DefaultValueTuplePackageVersion>4.3.1</DefaultValueTuplePackageVersion>
+    <DefaultFSharpPackageVersion>4.3.3</DefaultFSharpPackageVersion>
```

Comment:
Suppose we'll need to add this to list of things to update when we rev FSharp.Core :(

Reply:
Indeed.  I expect we'll have a script do this.

Reply:
The package is not yet published, tomorrow, it will be published and dev15.6 will be merged into dev15.7, we will update this.  I'm guessing we won't have pulled this by then anyway.  So I expect we will have fix that value when we address the merge conflict, we will certainly get.

Reply:
Nah, we'll all just go perfectly mad updating the versions manually, because we can surely handle it without issue and ph'nglui mglw'nafh Cthulhu R'lyeh wgah'nagl fhtagn

Script sounds great!

Reply:
Also @brettfo has a clever idea to test this, tomorrow he will educate me, and I will add some tests.